frontend/components/commentView.tsx

```diff
@@ -0,0 +1,32 @@
+// import Image from 'next/image'
+import AnonPill, { NounSet } from "./anonPill";
+
+interface ICommentViewProps {}
+
+const CommentView: React.FC<ICommentViewProps> = ({}) => {
+  return (
+    <div className="bg-white rounded-md shadow-sm max-w-xl mx-auto py-4 px-5 border border-gray-200">
+      <div className="flex justify-between">
+        <div className="flex items-center space-x-2">
+          {/* <Image width={32} height={32} src="/public/nouns.png" alt="" /> */}
+          <img
+            className="inline-block h-8 w-8 rounded-full"
+            src="/punk-4156.jpeg"
+            alt=""
+          />
+          <p className="text-gray-800 font-semibold">Demis G</p>
```

Comment:
should we have some notion of names? if it's anon, presumably we don't want people to identify themselves beyond message contents (?)

also probably don't want the profile pic on comments to be a specific noun?